src/library_eventloop.js

```diff
@@ -149,12 +149,21 @@ LibraryJSEventLoop = {
     clearInterval(id);
   },
 
+  $registerPostMainLoop: (f) => {
```

Comment:
Maybe add a comment that as an optimization, this does not include MainLoop but instead checks for it at runtime and does nothing?

Separately, I wonder if we can do better than this. We could in theory add a "weak linking" mechanism, that would add some code only if MainLoop was actually included. Though if it's complicated it might not be worth it.

Reply:
Sadly I don't think we have such a weak linking mechanism today.  The way I did it here I think is the closest thing we have to weak linking in JS code.